boards/st/stm32l1_disco/board.yml

```diff
@@ -1,5 +1,9 @@
-board:
-  name: stm32l1_disco
-  vendor: st
-  socs:
-    - name: stm32l151xb
+boards:
+  - name: stm32l1_disco
+    vendor: st
+    socs:
+      - name: stm32l151xb
+  - name: stm32l152c_disco
+    vendor: st
+    socs:
+      - name: stm32l152xc
```

Comment:
@tejlmand What is your advice on this ? Should we rather split in 2 different boards, instantiate revisions, keep current proposal of exposing 2 variants of the board reference in this same folder

Reply:
Hi @erwango and @tejlmand ,

This change follows how mimxrt1060_evk (boards/nxp/mimxrt1060_evk) handles two variants in the same folder. However, if we decide to choose another way, it is fine from my side.